compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3"
+
+services:
+  api:
+    container_name: api
+    restart: on-failure
+    build: .
```

Comment:
If you have a docker hub thingy, you should address the image here

And you should publish the image which you generate with your dockerfile, to that docker hub thingy